src/components/styles/BookAdd.css

```diff
@@ -1,7 +1,45 @@
-.book-add {
+.book-add * {
+  font-family: 'Montserrat', sans-serif;
+}
+
+.add {
+  font-size: 1.25rem;
+  opacity: 0.5;
+  margin-bottom: 20px;
+}
+
+form {
   display: flex;
+  justify-content: space-between;
+}
+
+form input::placeholder {
+  padding-left: 15px;
+  opacity: 0.5;
+}
+
+.book-title {
+  padding-left: 35px;
+  width: 50%;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+}
+
+.book-author {
+  padding-left: 25px;
+  width: 22%;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
 }
 
-.book-add button {
-  padding: 10px 30px;
+.btn-add {
```

Comment:
- [ ] [REQUIRED] Great job with the styling I must say 👍🏽 However, all elements in the design should be added. You have only added just one `Bookcard`. Kindly implement the other two cards.
- Also, there is an icon right at the top of the design, kindly add it also to your design.
### Your design
![Screenshot from 2022-04-02 14-56-59](https://user-images.githubusercontent.com/58692590/161386976-901e18e6-cfc8-4ad2-8c10-04536f66bca2.png)

### Design Template
![Screenshot from 2022-04-02 14-55-26](https://user-images.githubusercontent.com/58692590/161387027-556d4016-b459-4156-a6fe-a816f6c603d4.png)